packages/paste-style-props/src/types/typography.ts

```diff
@@ -38,6 +42,10 @@ export type OverflowWrap = ResponsiveValue<OverflowWrapOptions>;
 export type TextDecoration = ResponsiveValue<TextDecorationOptions>;
 export type TextOverflow = ResponsiveValue<TextOverflowOptions>;
 export type TextTransform = ResponsiveValue<TextTransformOptions>;
+export type FontVariantNumeric = ResponsiveValue<FontVariantNumericProperty>;
+export type ListStyleImage = ResponsiveValue<ListStyleImageProperty>;
+export type ListStylePosition = ResponsiveValue<ListStylePositionProperty>;
+export type ListStyleType = ResponsiveValue<ListStyleTypeProperty>;
```

Comment:
Pulled in a few generic typography related css property types so type can be control from one location